### PR TITLE
3.1.0 changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Release notes updated

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
-    - The bot now has increased access to various modules, maintaining privacy is paramount. Sensitive data is still being actively masked for all interactions with the LLM.
+### Key Highlights
+- **Press and Hold**: Activate the voice input by pressing and holding the microphone icon.
+- **Automatic Stop**: Voice recording stops automatically when the microphone is released, simplifying the input process.
+- **Quick Send**: After recording, press **Enter** to send the response for processing.
 
-    - The bot can now hold conversations to understand requirements and generate a more targeted playbook.
+> [!Note]
+> This feature is currently unsupported on the Firefox browser.
```
Comment (cs-suyog-jinturkar) [inline release_notes.md]: please add this note in the content/setup section as well.

File: release_notes.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
-    - Through enhanced comprehension, the bot can also navigate to modules, its records, and playbook logs; in addition to alert investigation, remediation, escalation to incident, and even setting up war rooms, among several others.
-
-    - The bot now asks questions if it needs more information or context to perform a task. For example, in response to block an indicator, it asks for the indicator's value, type, and reason for blocking.
-
-- Previous conversations remain in the conversation window and help provide context to the bot, until users clear the conversation. The conversations persist throughout repeated logins and logouts.
\ No newline at end of file
+Try the Voice Recognition feature to elevate your SOC workflow!
```
Comment (cs-suyog-jinturkar) [inline release_notes.md]: Clear conversation option now doesn't delete the conversation thread from the openAI. more details can be provided by @cs-ankit-gupta

Comment (cs-suyog-jinturkar) [reply release_notes.md]: applicalbe in release notes as well in content/usage section

File: docs/usage.md [hunk 3]
```diff
@@ -16,7 +18,28 @@ Click the icon to open the The **FortiAI** bot dialog to begin your interaction
 To manually clear a conversation in the bot, click the **Clear Conversation** button.
 
 >[!Note]
->Clearing a conversation deletes the thread and hence generative AIs, like OpenAI, may lose context.
+>Clearing a conversation deletes the conversation and starts a new thread. Hence generative AIs, like OpenAI, may lose context.
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: Now it does not delete a thread @cs-abhishek-shukla, we only clear the conversation

File: release_notes.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
 # What's New
 
-- The AI Assistant can now understand and respond to context sensitive conversations and perform given SOC tasks.
+**Enhanced AI Assistant with Voice Recognition Feature**
```
Comment (cs-ankit-gupta) [inline release_notes.md:3]: New release note as that now we are not deleting the thread from OpenAI, we are only clearing the conversation

**Review by cs-ankit-gupta — CHANGES_REQUESTED** [review]

File: docs/usage.md [hunk 5]
```diff
@@ -16,7 +18,28 @@ Click the icon to open the The **FortiAI** bot dialog to begin your interaction
...
 
 >[!Note]
->Clearing a conversation deletes the thread and hence generative AIs, like OpenAI, may lose context.
+>Clearing a conversation starts a new thread while retaining the old thread in OpenAI. However, since a new thread is created, generative AIs, like OpenAI, may lose context.
+
+## Voice Recognition
+
+Use voice commands to generate playbooks or perform myriad of tasks. This section explains how to use the bot's voice recognition feature
+
+1. Click to launch a relevant module for the bot icon to appear. For example, *Alerts*, *Incidents*, and *Playbooks* are some modules on which the bot icon appears.
```
Comment (cs-rahul-vishwakarma) [inline docs/usage.md]: "Playbooks" is not a module. Change it to "Indicators"

**Review by cs-rahul-vishwakarma — CHANGES_REQUESTED** [review]

**Pull request merged**